### PR TITLE
UI: use deepexi-theme and adjust sidebar

### PR DESCRIPTION
**Pull request by Barretem**
This PR main update UI template.

### CODE REVIEW

File: src/store/index.js [hunk 1]
```diff
@@ -34,6 +36,8 @@ export const mutations = {
         path: cookiePath
       })
     })
+    state.mainHeadTitle = ''
```
Comment (levy9527) [inline src/store/index.js]: 这个可以去掉

File: src/plugins/element.js [hunk 2]
```diff
@@ -70,7 +74,10 @@ Vue.use(Dialog)
 Vue.use(Tooltip)
 // Vue.use(Cascader)
 // Vue.use(Autocomplete)
-
+Vue.use(Scrollbar)
+Vue.use(Breadcrumb)
+Vue.use(BreadcrumbItem)
+Vue.use(Badge)
```
Comment (levy9527) [inline src/plugins/element.js]: 局部使用

File: src/container/side-bar/index.vue [hunk 3]
```diff
@@ -0,0 +1,246 @@
...
+  }
+}
+</script>
+
+<style lang="less">
+@import '../../assets/variables.less';
+
+@title-bg: rgba(242, 244, 249, 1);
+@menu-height: 50px;
+@primary-color: #1890ff;
```
Comment (levy9527) [inline src/container/side-bar/index.vue]: 尽量变量集中在一起

**Comment by Barretem** [comment]
Everything has been modified as required. Please spending time to code review. Thanks

**Pull request merged**